config/initializers/secret_token.rb

```diff
@@ -1,7 +1,10 @@
+# The values in this file are overwritten when deploying
+
 # Be sure to restart your server when you modify this file.
 
 # Your secret key for verifying the integrity of signed cookies.
 # If you change this key, all old signed cookies will become invalid!
 # Make sure the secret is at least 30 characters and all random,
 # no regular words or you'll be exposed to dictionary attacks.
-SmartAnswers::Application.config.secret_token = '8b5eb781d13b5968e41f62ac831d9c729f84983cb3bb29ab03f77c2b4948c9a679198f0d4111a825ad86607b82cd4e291c3bef35d86da04109f6d6799cc8e69e'
+SmartAnswers::Application.config.secret_token = 'replacedondeploy'
+SmartAnswers::Application.config.secret_key_base = 'replacedondeployaswell'
```

Comment:
I think this will have to be added to the deployment script. Can someone who has access to it make a PR for that please? /cc @dsingleton @bradleywright